examples/air-quality-sensor-app/linux/main.cpp

```diff
@@ -52,7 +52,14 @@ int main(int argc, char * argv[])
 {
     VerifyOrDie(ChipLinuxAppInit(argc, argv) == 0);
 
-    std::string path = kChipEventFifoPathPrefix + std::to_string(getpid());
+    const char * app_id = std::getenv("CHIP_AIR_QUALITY_APP_ID");
+    if (app_id == nullptr)
+    {
+        app_id = "54321";
+    }
+
+    std::string app_id_str = app_id;
+    std::string path       = kChipEventFifoPathPrefix + app_id_str;
```

Comment:
```suggestion
    std::string path = kChipEventFifoPathPrefix + std::string(app_id);
```

This option is leaner, and it might give the compiler an opportunity to optimize the object code.

Try this on the other `.cpp` files as well.